src/pages/getting-started/app-guides/chat-webhooks/index.mdx

```diff
@@ -3,107 +3,127 @@ weight: 70
 
-1. Create an app in <Placeholder id="DEVELOPER_CONSOLE_URL" />.
-2. Set up the App Authorization building block.
-3. Handle direct installation and authorization in your app.
-4. Configure the Chat webhooks building block.
+As a developer, you need to handle the authorization flow in your app. For a backend app, we recommend [Authorization code grant](/authorization/authorizing-api-calls/#authorization-code-grant), where your app exchanges `code` for `token`.
 
-#### Step 1 Create an app
+### Step 3: Configure chat webhooks
 
-This is a one-step process as you only need to give your app a name. You don't need to configure the Agent App Widget building block if your application has no frontend side and only runs in the background.
+Configuring chat webhooks in Developer Console is equivalent to calling the [Register Webhooks](/management/configuration-api/#register-webhook) method from the Configuration API. However, making use of the Chat Webhooks building block is **much more convenient** because you don't need to support [enabling webhooks](/management/configuration-api/#enable-license-webhook) – **LiveChat will do that for you** when someone installs you app.
 
-#### Step 2 Set up the App Authorization block
+The configuration of the Chat Webhooks building block is a three-step process and involves the following aspects:
 
-Why is it necessary? You're building an application that, once installed, will call the Configuration API to register webhooks for a user's license. To perform this operation, the user needs to authorize your application first.
+- [Webhook URL](#webhook-url)
+- [Secret key](#secret-key)
+- [Webhook triggers](#webhook-triggers)
 
-In short:
+#### Webhook URL
 
-- Add the right scopes. For [registering webhooks](/management/configuration-api/v3.2/#register-webhook), you need `webhooks--my:rw`, but add all the scopes you'll need, depending on the functionalities of your app.
-- Provide a **Direct installation URL** and add it to the Redirect URI whitelist.
+All the webhooks will be sent to the URL you provide in the **Webhook URL** field. That's likely the address of your backend service, but you can use [https://webhook.site](https://webhook.site) for testing.
 
-💡 We have a [detailed tutorial](/getting-started/authorization/authorization-in-practice/) on how to configure the App Authorization building block for a server-side app.
+#### Secret key
 
-#### Step 3 Handle direct installation and authorization in your app
+We'll include your **secret key** in the payload of each webhook. This way, your application can verify the webhooks came from us. Make sure you save the generated secret key as it will disappear after you refresh the page. Also, if you change something in the building block configuration, you'll need to regenerate the secret key.
```

Comment:
`you'll need to regenerate the secret key.` - developers can insert their old secret as well, they dont need to regenerate it